cjs/index.cjs

```diff
@@ -2,7 +2,13 @@ const getExport = name => import("../dist/index.mjs").then(r => r[name]);
 const createCaller = name => (input, init) => getExport(name).then(function_ => function_(input, init));
 
 exports.fetch = createCaller("fetch");
+
 exports.ofetch = createCaller("ofetch");
+exports.ofetch.create = defaultOptions => getExport("ofetch").then($fetch => $fetch.create(defaultOptions));
+exports.ofetch.raw = (input, init) => getExport("ofetch").then($fetch => $fetch.raw(input, init));
+exports.ofetch.native = (input, init) => getExport("ofetch").then($fetch => $fetch.native(input, init));
+
```

Comment:
Since `ofetch` is actually `$fetch`, I added these exports.

https://github.com/unjs/ofetch/blob/5943c5fcb61c2a55ef0ad11852541a7765e55c55/src/index.ts#L20